Lib/test/test_tempfile.py

```diff
@@ -848,6 +848,12 @@ def test_for_tempdir_is_bytes_issue40701_api_warts(self):
         finally:
             tempfile.tempdir = orig_tempdir
 
+    def test_absolute_path(self):
+        """Test that the path returned is absolute"""
```

Comment:
Minor: use a comment rather than a docstring (see other test methods higher); use a better test name (test absolute path makes us think that the input is an absolute path, but here it’s rather something like test path returned always absolute)

Reply:
Done in ff507d9002